alica_tests/src/test/test_alica_authority.cpp

```diff
@@ -104,6 +105,8 @@ TEST(AllocationDifference, MessageCancelsUtil) {
 }
 
 TEST_F(AlicaEngineAuthorityManager, authority) {
+    ASSERT_EQ(setjmp(restore_point), 0);
```

Comment:
This exposes way too much internals.
For the case without fixture you can define a macro that calls `TEST`.

With fixture, it is a little more complicated. But at the very least you can provide a macro `ASSERT_NO_TRAP` (or think of a better name).